modules/security_hub/variables.tf

```diff
@@ -15,3 +20,19 @@ variable "region" {
   default     = "eu-west-1"
   description = "The name of the AWS region where SecurityHub will be enabled"
 }
+
+variable "sns_endpoint" {
+  type        = string
+  description = "Endpoint for SNS topic subscription"
+}
+
+variable "sns_endpoint_protocol" {
+  type        = string
+  description = "Endpoint protocol for SNS topic subscription"
+}
+
+variable "sns_security_topic_subscription" {
+  type        = bool
+  default     = false
+  description = "Enable SNS aggregated security topic subscription"
+}
```

Comment:
Could these values be specified in a single `sns_security_subscription` object?

Reply:
done